### PR TITLE
Merges Feature/ActiveSession to Develop

### PR DESCRIPTION
**Pull request by pedrocerrano**
Feature/ActiveSession refactors files from ModalHome to NewSession.

### CODE REVIEW

File: MapShare/MapShare/UI/Active Session View/ActiveSessionViewController.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    
+    
+    //MARK: - PROPERTIES
+    var activeSessionViewModel: ActiveSessionViewModel!
+    
+    
+    //MARK: - LIFECYCLE
+    override func viewDidLoad() {
+        super.viewDidLoad()
+//        activeSessionViewModel = ActiveSessionViewModel()
```
Comment (maxpoff) [inline MapShare/MapShare/UI/Active Session View/ActiveSessionViewController.swift]: Is this intentionally left in? Unused code should be removed

File: MapShare/MapShare/UI/Active Session View/Custom Views/ActiveSessionTableViewCell.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    @IBOutlet weak var isOrganizerImageView: UIImageView!
+    @IBOutlet weak var dotColorImageView: UIImageView!
+    
+    
+    //MARK: - FUNCTIONS
+    func configureCell(with member: Member) {
+        memberNameLabel.text = member.memberName
+        if member.isOrganizer == false {
+            isOrganizerImageView.isHidden = true
+        }
```
Comment (maxpoff) [inline MapShare/MapShare/UI/Active Session View/Custom Views/ActiveSessionTableViewCell.swift:23]: Fix extra space on the following few lines

**Review by maxpoff — APPROVED** [review]
Address comments and you're good to go

File: MapShare/MapShare/UI/Active Session View/ActiveSessionViewController.swift (hunk 1 above)
Comment (pedrocerrano) [reply MapShare/MapShare/UI/Active Session View/ActiveSessionViewController.swift]: Yes, it was a note to myself and my partner that the ViewModel would need to be initialized, however I knew that I would handle that via segue Navigation.

In future cases, I'll use my #warning("")

**Pull request merged**